pkg/image/registryclient/client.go

```diff
@@ -26,7 +26,6 @@ import (
 	"github.com/docker/distribution/registry/client/auth/challenge"
 	"github.com/docker/distribution/registry/client/transport"
 	"github.com/opencontainers/go-digest"
-	imagereference "github.com/openshift/library-go/pkg/image/reference"
```

Comment:
This commit merged already - surprised it still shows up here

Reply:
i cleaned up the commits & cherry-picks, it's no longer there